src/routes/verify/[verifyToken]/+page.svelte

```diff
@@ -0,0 +1,91 @@
+<script>
+    // Framework imports
+    import { onMount } from "svelte";
+    import { urls, tryFetchingRestricted, toaster } from "$lib/util.js";
+    import { getToastStore, ProgressRadial } from "@skeletonlabs/skeleton";
+    import { goto } from "$app/navigation";
+
+    // Definitions
+    // Constants
+    const toastStore = getToastStore();
+
+    // Toast settings
+    const successToast = {
+        message: "Verifizierung erfolgreich",
+        bg: "primary",
+    };
+
+    const errorToast = {
+        message: "Verfizierung fehlgeschlagen",
+        bg: "error",
+    };
+
+    const serverErrorToast = {
+        message: "Fehler auf der Serverseite",
+        bg: "error",
+    };
+    
+    // Variables
+    export let data;
+
+    // Initialize verfied on default empty promise
+    let verified = new Promise(async (resolve, reject) => {});
+
+    // Functions
+    async function verifyTheToken() {
+        console.log(data.verifyToken);
+        verified = new Promise(async (resolve, reject) => {
+            try {
+                const response = await tryFetchingRestricted(
+                    urls.get.verifyUserToken + data.verifyToken,
+                );
+                if (response.ok) {
+                    resolve(await response);
+                    toastStore.trigger(toaster(successToast));
+                    goto("/login");
+                } else if (response.status == 400) {
+                    toastStore.trigger(toaster(errorToast));
+                    throw new Error("Couldnt be verified");
+                } else {
+                    toastStore.trigger(toaster(serverErrorToast));
+                    throw new Error("Server error");
+                }
+            } catch (err) {
+                reject(err); // Rethrow so Svelte can handle it
```

Comment:
Den Toast bitte in das catch machen, sodass dieser auch bei backend nicht erreichbar angezeigt wird.
```suggestion
                    throw new Error("Server error");
                }
            } catch (err) {
                toastStore.trigger(toaster(serverErrorToast));
                reject(err); // Rethrow so Svelte can handle it
```

Reply:
hab ich grad ausprobiert, aber dann bekommt man 2 toast wenn das backend 400 also nicht verfiziert zurück gibt. Der 2. toast soll ja eigentlich nur dann kommen, wenn ein anderer Fehlercode zurückkommt

Reply:
obwohl, i guess kann man machen